platformio.ini

```diff
@@ -314,14 +314,13 @@ lib_deps = ${esp32.lib_deps}
 board_build.partitions = ${esp32.default_partitions}
 
 [env:esp32s2_saola]
-board = esp32dev
-board_build.mcu = esp32s2
-platform = espressif32
+board = esp32-s2-saola-1
+platform = https://github.com/tasmota/platform-espressif32/releases/download/v3.4.1/Tasmota-platform-espressif32.zip
```

Comment:
Maybe you could try this as platform.
https://github.com/tasmota/platform-espressif32/releases/download/v2.0.2idf/platform-espressif32-2.0.2.zip

Reply:
Diner time but yeah I'll try this in a bit

Reply:
Ehh no real difference from changing platforms

Reply:
tried another board fresh out the bag and same

```
rst:0x1 (POWERON),boot:0x8 (SPI_FAST_FLASH_BOOT)
SPIWP:0xee
mode:DIO, clock div:1
load:0x3ffe6100,len:0x510
load:0x4004c000,len:0xa50
load:0x40050000,len:0x28bc
entry 0x4004c18c
./components/esp_littlefs/src/littlefs/lfs.c:1071:error: Corrupted dir pair at {0x0, 0x1}
E (284) esp_littlefs: mount failed,  (-84)
E (285) esp_littlefs: Failed to initialize LittleFS
[   749][E][esp32-hal-misc.c:125] disableCore0WDT(): Failed to remove Core 0 IDLE task from WDT
[  5149][E][vfs_api.cpp:102] open(): /littlefs/presets.json does not exist, no permits for creation
[  5173][E][vfs_api.cpp:102] open(): /littlefs/wsec.json does not exist, no permits for creation
[  5247][E][vfs_api.cpp:102] open(): /littlefs/ledmap.json does not exist, no permits for creation
Ada
```